web/src/lib/components/map-page/map-settings-modal.svelte

```diff
@@ -35,6 +35,7 @@
       <SettingSwitch title="Only favorites" bind:checked={settings.onlyFavorites} />
       <SettingSwitch title="Include archived" bind:checked={settings.includeArchived} />
       <SettingSwitch title="Include shared with me" bind:checked={settings.withPartners} />
+      <SettingSwitch title="Include shared albums" bind:checked={settings.withSharedAlbums} />
```

Comment:
Should we remember this other option so it is more obvious what the difference is?

Reply:
I renamed `Include shared with me` to `Include shared from partners`. Another option would be `Include assets from partners`.

What do you think, which option sounds better? Or do you have other suggestions?

Reply:
I'm not Jason but I would suggest `Include partner assets` or `Include shared partner assets`. Also, this rename is important for clarity.

Reply:
I renamed it to `Include shared partner assets` now, thanks for the suggestion @NicholasFlamy